modules/doCheckIn.js

```diff
@@ -0,0 +1,45 @@
+const fetch = require('isomorphic-fetch');
+const endpoints = require('../helpers/endpoints.js'); //For Creating shorter URL's in this Module
+const { getConfig } = require('../index');
+
+/**
+ * Loads Favorite Members
+ * @param {SecurityString} sid For authenticating with the Narvii-API.
+ * @param {String} coms A list of communities to get the check-in status, separated by "&"
+ * @param {String} timezone The desired timezone to check(It will affect the user Check-In Functionality)
+ * @returns {Object} Object with all the Check In info of the communities specified from the Logged-in User has in an Array.
+ */
+
+module.exports = async function doCheckIn(com, timezone) {
+    let checkInData = {
```

Comment:
It would be better if we can central all of ours default output object inside a helper class call `object.js`

Reply:
I had set it inside the function but it was temporary. I forgot to migrate it to the ``object.js``
I will be doing and update the PR

Reply:
Oh silly me 🙈 

Thanks a lot @akatsukilevi 😄

Reply:
https://github.com/AminoJS/Amino.JS/pull/46/commits/04be7c049245b1a2c56e4c7d280539ec01169ca6 fixes this